src/main/java/org/osiam/storage/query/ScimExpression.java

```diff
@@ -23,22 +23,28 @@
 
 package org.osiam.storage.query;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Locale;
+
 /**
  * Representation of scim search expression.
  */
 public class ScimExpression {
 
-    private final String field;
+    private static final Logger LOG = LoggerFactory.getLogger(ScimExpression.class);
```

Comment:
I think [we once agreed](https://github.com/osiam/osiam/issues/40) on how we name loggers.